compiler/vm/vm.nim

```diff
@@ -428,57 +628,66 @@ proc cleanUpOnReturn(c: PCtx; f: PStackFrame): int =
       discard f.safePoints.pop
       return pc + 1
 
-proc opConv(c: PCtx; dest: var TFullReg, src: TFullReg, desttyp, srctyp: PType): bool =
+template strVal*(r: TFullReg): untyped =
+  {.line.}: 
```

Comment:
I don't think you need this pragma. It kinda does nothing if you don't give it a parameter.

Reply:
The manual says (in the section 'line pragma'):
```
If the line pragma is used with a parameter, the parameter needs be a tuple[filename: string, line: int].
If it is used without a parameter, system.InstantiationInfo() is used
```

I used it here so that the "calling" location is shown in stack-traces when the assertion fails

Reply:
I actually didn't know about that feature :P

All is fine then.